Gemfile

```diff
@@ -61,3 +61,5 @@ gem 'rubocop', '>= 1.0', '< 2.0'
 gem 'rails-controller-testing'
 
 gem 'devise'
+
```

Comment:
- [ ] You should install [rswag](https://github.com/rswag/rswag) gem

Reply:
- [ ] You should have an integration spec to describe and test the existing API endpoints. Kindly checkout the [example](https://github.com/rswag/rswag#:\~:text=Create%20an%20integration%20spec%20to%20describe%20and%20test%20your%20API.%20There%20is%20also%20a%20generator%20which%20can%20help%20get%20you%20started%20rails%20generate%20rspec%3Aswagger%20API%3A%3AMyController) provided in the documentation.

Reply:
Hi @GzimAsani. Thanks for the review but unfortunately you have the wrong requirements, [these are the requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_add_authorization.md) for today's project.

Reply:
Hi @sevinchek. I think you asked for a code review and selected a different project when you did so. Please ask for another CR. :laughing:

Reply:
Hi @GzimAsani. It's the correct project. As you can see:
![imagen](https://user-images.githubusercontent.com/39852288/141013230-345aeab2-ff82-40f5-b845-6dc947383ffa.png)
Maybe reviewers have a wrong link.

Reply:
- [ ] That is interesting because in the requirements that I was getting they were different. Anyhow I apologize and please ask for another code review

Reply:
Ok. I asked for another review but we lost a review attempt.